packages/jobs/lib/crons/autoIdleDemo.ts

```diff
@@ -0,0 +1,128 @@
+import { schedule } from 'node-cron';
+import {
+    CommandToActivityLog,
+    ErrorSourceEnum,
+    ScheduleStatus,
+    SyncClient,
+    SyncCommand,
+    SyncConfig,
+    createActivityLog,
+    createActivityLogMessageAndEnd,
+    db,
+    errorManager,
+    updateSuccess as updateSuccessActivityLog,
+    updateScheduleStatus,
+    isErr
+} from '@nangohq/shared';
+import tracer from 'dd-trace';
+
+export async function cronAutoIdleDemo(): Promise<void> {
+    schedule('1 * * * *', () => {
+        const span = tracer.startSpan('cron.syncs.idleDemo');
+        tracer.scope().activate(span, async () => {
+            try {
+                await exec();
+            } catch (err: unknown) {
+                const e = new Error('failed_to_auto_idle_demo', { cause: err instanceof Error ? err.message : err });
+                errorManager.report(e, { source: ErrorSourceEnum.PLATFORM }, tracer);
```

Comment:
is it worth reporting to sentry? can we just `span.setTag('error', err)`

Reply:
I don't see why not? The discussion is still open wether or not we are keeping Sentry but for the meantime I think it's relevant, wdyt?